### PR TITLE
Fix minor layout issues in responsive rtl

### PR DESCRIPTION
**Pull request by fnaysee**

### CODE REVIEW

File: publishable/assets/css/rtl.css [hunk 1]
```diff
@@ -276,6 +276,16 @@
     float: right;
 }
 
+[dir="rtl"]  .voyager .navbar .navbar-nav .dropdown-menu, [dir="rtl"] .voyager .navbar.navbar-default .navbar-nav .dropdown-menu {
```
Comment (fletch3555) [inline publishable/assets/css/rtl.css]: The second selector is not necessary here as it's a subset of the first (.navbar vs .navbar.navbar-default) and therefore adds no value.

File: publishable/assets/css/rtl.css [hunk 2]
```diff
@@ -276,6 +276,16 @@
...
+    left: 0;
+    right: auto !important;
+    position: absolute !important;
+}
+
+[dir="rtl"]  .voyager .navbar .navbar-nav li.dropdown.open .dropdown-menu, [dir="rtl"] .voyager .navbar.navbar-default .navbar-nav li.dropdown.open .dropdown-menu {
```
Comment (fletch3555) [inline publishable/assets/css/rtl.css]: Same as above.

**Review by fletch3555 — CHANGES_REQUESTED** [review]

**Comment by fletch3555** [comment]
Please avoid closing one PR and open another just to add changes.  PRs are a live comparison of the two branches, so they will automatically update when you push new commits.

File: npm-debug.log [hunk 3]
```diff
@@ -0,0 +1,49 @@
+0 info it worked if it ends with ok
```
Comment (fletch3555) [inline npm-debug.log]: This file should not be committed.  Please be cognizant of what you're committing.

**Review by fletch3555 — CHANGES_REQUESTED** [review]

**Comment by fnaysee** [comment]
@fletch3555 Sry, i'm new on requesting pulls. i was created another branch and after that created another branch with better name!
Is it ok if i commit all changes to code or styles in this branch and current pr ?

**Comment by fletch3555** [comment]
No worries.  Don't worry about branch names (pick something somewhat reasonable, but there's no need to change it after it got this far).

For the other changes, you should limit pull requests to distinct functionality, and since a PR is only a single branch, you should limit your changes to distinct branches.  For example, if you're making a bunch of code-style/formatting/spelling/etc changes, then they can all go together in one branch/PR.  If you're fixing a bug in the BREAD system, and also adding a new language worth of translation files, then those should be 2 separate branches/PRs.  Make sense?

**Comment by fnaysee** [comment]
OK got it.

**Review by fletch3555 — APPROVED** [review]

**Comment by fletch3555** [comment]
@wp-src, looks like you deleted your fork, so we're unable to merge this in.  If you'd like to recreate the PR, we'll be happy to get it merged in.

**Comment by emptynick** [comment]
Closing.
If someone wants to recreate the changes made in this PR feel free to do so.

**Pull request closed without merging**